### ISSUE TITLE
[Bug]: Config called too early

### ISSUE THREAD

**Issue by SiverDX** [Fuzss/airhop#13]
### Mod Loader (Required)

Forge

### Minecraft Version (Required)

1.19.2

### Mod Version (Required)

4.2.1

### Notes (Required)

Caused by

https://github.com/Fuzss/airhop/blob/346c78ec128f5ae71b136e447d333d8d0c310653/Common/src/main/java/fuzs/airhop/enchantment/AirHopEnchantment.java#L30-L34

```
[18Nov2023 15:06:12.449] [Worker-ResourceReload-8/ERROR] [Puzzles Lib/]: Calling server config when it is not yet available! This is a bug! Message: Config data is missing
java.lang.Exception: Config not yet available
	at fuzs.puzzleslib.config.ForgeConfigDataHolderImpl.lambda$testAvailable$1(ForgeConfigDataHolderImpl.java:46) ~[PuzzlesLib-v4.4.3-1.19.2-Forge.jar%23776!/:4.4.3]
	at com.mojang.datafixers.util.Either$Right.ifRight(Either.java:109) ~[datafixerupper-5.0.28.jar%23121!/:?]
	at fuzs.puzzleslib.config.ForgeConfigDataHolderImpl.testAvailable(ForgeConfigDataHolderImpl.java:45) ~[PuzzlesLib-v4.4.3-1.19.2-Forge.jar%23776!/:4.4.3]
	at fuzs.puzzleslib.config.ConfigDataHolderImpl.getOrCreateDefaultConfig(ConfigDataHolderImpl.java:70) ~[PuzzlesLib-v4.4.3-1.19.2-Forge.jar%23776!/:4.4.3]
	at fuzs.puzzleslib.config.ConfigDataHolderImpl.config(ConfigDataHolderImpl.java:59) ~[PuzzlesLib-v4.4.3-1.19.2-Forge.jar%23776!/:4.4.3]
	at fuzs.puzzleslib.config.ConfigHolder.get(ConfigHolder.java:28) ~[PuzzlesLib-v4.4.3-1.19.2-Forge.jar%23776!/:4.4.3]
	at fuzs.airhop.enchantment.AirHopEnchantment.m_6586_(AirHopEnchantment.java:33) ~[AirHop-v4.2.1-1.19.2-Forge.jar%23526!/:4.2.1]
	at shadows.apotheosis.ench.EnchModule.getDefaultMax(EnchModule.java:453) ~[Apotheosis-1.19.2-6.4.1.jar%23533!/:6.4.1]
	at shadows.apotheosis.ench.EnchantmentInfo.load(EnchantmentInfo.java:69) ~[Apotheosis-1.19.2-6.4.1.jar%23533!/:6.4.1]
	at shadows.apotheosis.ench.EnchModule.reload(EnchModule.java:476) ~[Apotheosis-1.19.2-6.4.1.jar%23533!/:6.4.1]
	at shadows.apotheosis.ench.EnchModule.init(EnchModule.java:143) ~[Apotheosis-1.19.2-6.4.1.jar%23533!/:6.4.1]
	at shadows.apotheosis.ench.__EnchModule_init_FMLCommonSetupEvent.invoke(.dynamic) ~[Apotheosis-1.19.2-6.4.1.jar%23533!/:6.4.1]
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%23134!/:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:107) ~[javafmllanguage-1.19.2-43.3.0.jar%23885!/:?]
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$10(ModContainer.java:122) ~[fmlcore-1.19.2-43.3.0.jar%23884!/:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.scan(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) [?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) [?:?]
```

**Comment by Fuzss** [comment]
Thanks, that's just a note for me that the config value is still at it's default there. So nothing to worry about really.

**Comment by Fuzss** [comment]
It's fixed on 1.20+, so not really worth the hassle to release a new version for 1.19.2.
